### DISCUSSION TITLE
Command confirmation idea

### DISCUSSION THREAD

**Discussion by LogGits** [Zrips/CMI#223] [Feature Requests]
A confirm command section of the config. When the user types the command they will have to retype it before the time variable (in seconds) is up.

**EXAMPLE USAGE:**
```yaml
ConfirmCommand:
  - pay:20 #will ask them to confirm the /pay command within 20 seconds
  - clearinventory:50 #within 50seconds
  - ban:5 #within 5seconds
```

**Comment by WolverStones** [comment]
Great idea

**Comment by arturek1666** [comment]
I think I did say something similar before to zrips about adding an confirm ;)

**Comment by ghost** [comment]
Would love to see this as a GUI thing.

**Comment by mrfloris** [comment]
```
  # Determines if player needs to confirm money payment by clicking on chat message
  Confirmation: false
```
`/cmi money pay` has a confirmation toggle.

```
    Clear:
      # When set to true, /cmi clear comamnd will require confirmation for it to be finalized
      Confirmation: true
```
`/cmi clear` has a confirmation toggle.

And currently `/cmi ban` does not. 

Don't get me wrong, I understand you want a section and enter any cmi command. Just contributing that at least those 2 can be turned on already.
